src/Unused/Parser.hs

```diff
@@ -21,15 +21,7 @@ responseFromParse =
     fmap $ Map.fromList . map (second $ calculateLikelihood . resultsFromMatches) . groupBy tmTerm
 
 parseTermMatches :: Parser [TermMatch]
-parseTermMatches = do
-    tm <- many1 $ do
-        m <- parseTermMatch
-        void eol
-
-        return m
-    eof
-
-    return tm
+parseTermMatches = many1 parseTermMatch <* eof
```

Comment:
If you'd like to keep the `eol` in this parser, and not in `parseTermMatch` below, this line would be:

```haskell
many1 (parseTermMatch <* eol) <* eof
```